components/Pages/BrokerPortal/index.js

```diff
@@ -60,10 +62,11 @@ export const BrokerPortalPage = ({
 
   return (
     <div className="broker-portal">
+      <Loading isLoading={loading} className='loading-center' />
       <BrokerageHeader />
       <Container background="#FAFAFA" padding="0 0 55px">
         <Container maxWidth="1080px" margin="0 auto">
-          <Container className="px-60 pt-10">
+          <Container padding='8px 60px 0 60px'>
```

Comment:
We have top gap for breadcrumbs as `10px` on other pages. Let's update it here as well. Also you can write it shorter
```suggestion
          <Container padding='10px 60px 0'>
```